ast3/Custom/typed_ast.c

```diff
@@ -211,6 +211,30 @@ err_free(perrdetail *err)
     Py_CLEAR(err->filename);
 }
 
+// from Python/pythonrun.c
+node *
+Ta3Parser_SimpleParseStringFlagsFilename(const char *str, const char *filename,
+                                         int start, int flags)
+{
+    perrdetail err;
+    node *n = Ta3Parser_ParseStringFlagsFilename(str, filename,
+                            &_Ta3Parser_Grammar, start, &err, flags);
+    if (n == NULL)
+        err_input(&err);
+    err_free(&err);
+    return n;
+}
+
+/* update compiler and parser flags based on feature version */
+void
+update_typed_ast_flags(PyCompilerFlags *flags, int *iflags, int feature_version)
```

Comment:
It looks like this is shared between different files -- so it should have a `Ta3` (or probably `_Ta3`) prefix.